test/unit/codegen/codegen_llvm_ir.cpp

```diff
@@ -71,7 +78,7 @@ std::vector<std::shared_ptr<ast::Ast>> run_llvm_visitor_helper(
 
     SymtabVisitor().visit_program(*ast);
     SolveBlockVisitor().visit_program(*ast);
-    CodegenLLVMHelperVisitor(vector_width).visit_program(*ast);
+    CodegenLLVMHelperVisitor(vector_width, /*nmodl_inline*/ false).visit_program(*ast);
```

Comment:
```suggestion
    CodegenLLVMHelperVisitor(vector_width, /*nmodl_inline=*/false).visit_program(*ast);
```

Reply:
And I was wondering why `clang-format` complained about that. I forgot the `=` :facepalm: